R/get.R

```diff
@@ -0,0 +1,41 @@
+#' Retrieves participants present in secuTrialdata
+#'
+#' Given a secuTrialdata object, this function simply returns a list of participants.
+#' Information included are participant IDs and corresponding study centre information,
+#' if available.
+#'
+#' @param dat secuTrialdata object containing participant IDs and centre information
+#' @return data frame containing a list of participants present in dat
+#' @export
+#'
+#' @examples
+#'
+#' path <- system.file("extdata", "sT_exports", "export_options",
+#'                     "s_export_CSV-xls_CTU05_20191003-144349_all_info.zip",
+#'                     package = "secuTrialR")
+#' sT_export <- read_secuTrial(path)
+#'
+#' # show participants
+#' participants <- get_participants(sT_export)
+#'
+get_participants <- function(dat){
```

Comment:
It would be better to call this `get_patients` or `get_cases` since participants in secuTrial are technically data entry personnel and not people enrolled as study cases. Thus, the current name may cause some confusion.

Reply:
This was my suggestion. Not all people who participate in trials are patients or cases, but they are all participants, so from a clinical trials perspective this makes sense. Data entry people are not typically referred to as participants in the normal world... The normal user of secuTrialR will be statisticians or something like that, not a DM so probably wont be aware that secutrial uses the term in a different (wrong, IMO) sense

Reply:
I would suggest we leave it with this, but if anyone complains, we could re-consider. Note that the same is true for the subset_secuTrial() input parameter - currently, it's not "patient", but "participant".

Reply:
If both of you think its not an issue than we can leave it as is. However, I just got genuinely confused to whether it was retrieving patients/cases or participants. So I would still prefer the change. Up to you. Otherwise it can be merged from my side.

Reply:
For the record: We will leave it as is for now and will have a meeting to discuss naming and nomenclature in general to prepare for the release of a version 1.0.